R/probabilistic_segmentation.R

```diff
@@ -975,7 +982,7 @@ ploidetect_cna_sc <- function(all_data, segmented_data, tp, ploidy, maxpeak, ver
   common_maf_means <- testMAF(as.numeric(common_call), tp)
   maf_variance <- match_kde_height(as.numeric(unlist(unmerge_mafs(subcl_seg$maf[subcl_seg$CN == common_call]))), means = common_maf_means, sd = 0.03)
   
-  subcl_seg %>% filter(chr == "1") %>% ggplot(aes(x = pos, y = corrected_depth, color = factor(CN))) + geom_point() + scale_color_viridis(discrete = T)
+  #subcl_seg %>% filter(chr == "X") %>% ggplot(aes(x = pos, y = corrected_depth, color = factor(CN))) + geom_point() + scale_color_viridis(discrete = T)
   
```

Comment:
Generally, you should try not to have sections of code that are commented out.   Probably should remove lines completely or have something explaining it is not an accident or debug hold-over.

Was the 'subcl_seg' line commented out intentionally, and the "1" -> "X" is not relevant?

Reply:
This was a debugging plot that was held over, I should remove it at some point when I get the chance to do a cleanup of the code.